main_spec.tex

```diff
@@ -473,7 +473,28 @@ \subsection{Memory Ordering Routines}\label{subsec:memory_order}
 ensure ordering and/or delivery of memory store, blocking \OPR{Put}, \ac{AMO},
 and \OPR{put-with-signal}, as well as nonblocking \PUT{},
 \OPR{put-with-signal}, \GET{}, and \ac{AMO} routines to symmetric data
-objects.
+objects. Table~\ref{mem-order} lists the operations for which \openshmem
+memory ordering routines provide guarantees of ordering or completion.
+
+\begin{table}[h]
+  \begin{center}
+    \begin{tabular}{|l|c|c|}
+      \hline
+      Operations & Fence & Quiet \\ \hline
+      Memory Store                       & X & X \\
+      Blocking \OPR{Put}                 & X & X \\
+      Blocking \OPR{AMO}                 & X & X \\
+      Blocking \OPR{put-with-signal}     & X & X \\
+      Non-blocking \OPR{Put}             & X & X \\
+      Non-blocking \OPR{AMO}             & X & X \\
+      Non-blocking \OPR{put-with-signal} & X & X \\
+      Non-blocking \OPR{Get}             &   & X \\ \hline
+    \end{tabular}
+    \TableCaptionRef{List of operations ensured ordering and delivery by Memory
+    Ordering routines}
```

Comment:
"List of operations **that ensure ordering and/or** delivery by Memory Ordering routines."? Maybe another phrasing for the caption?